packages/xo-web/src/common/xo/index.js

```diff
@@ -1141,7 +1141,11 @@ export const deleteTemplates = templates =>
     await Promise.all(
       map(resolveIds(templates), id =>
         _call('vm.delete', { id }).catch(reason => {
-          if (vmIsTemplate.is(reason)) {
+          const template = getObject(store.getState(), id)
+          if (
+            forbiddenOperation.is(reason) &&
+            (template.other.default_template === 'true' || template.is_default_template)
```

Comment:
You can test the message instead.

Reply:
I had think to test the message directly but, it can make some issues of maintainability. For example if we decide to change the text of the error thrown. By testing directly  fields of the template we avoid this problem.

Reply:
You're right :+1: